rubygem-uuid/rubygem-uuid.spec

```diff
@@ -0,0 +1,86 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+# Generated from uuid-2.3.8.gem by gem2rpm -*- rpm-spec -*-
+%global gem_name uuid
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 2.3.8
+Release: 1%{?dist}
+Summary: UUID generator
+Group: Development/Languages
+License: MIT
+URL: http://github.com/assaf/uuid
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+Requires: %{?scl_prefix}rubygem(macaddr) >= 1.0
+Requires: %{?scl_prefix}rubygem(macaddr) < 2.0
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildArch: noarch
+
+%description
+UUID generator for producing universally unique identifiers based on RFC 4122
+(http://www.ietf.org/rfc/rfc4122.txt).
+
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{pkg_name}.
+
+%prep
+%{?scl:scl enable %{scl} "}
+gem unpack %{SOURCE0}
+%{?scl:"}
+%setup -q -D -T -n  %{gem_name}-%{version}
+%{?scl:scl enable %{scl} "}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:"}
+
+%build
+# Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
+gem build %{gem_name}.gemspec
+%{?scl:EOF}
+
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install -n %{SOURCE0}
+%{?scl:EOF}
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+
+mkdir -p %{buildroot}%{_bindir}
+cp -pa .%{_bindir}/* \
+        %{buildroot}%{_bindir}/
+
+find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
+
+%files
+%dir %{gem_instdir}
+%{_bindir}/uuid
```

Comment:
I don't think we should ship this as it'll conflict with the `uuid` package (https://apps.fedoraproject.org/packages/uuid/). Consider leaving out the binary script or renaming it.